site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md

```diff
@@ -0,0 +1,1001 @@
+
+{% Aside %}
+For the diagram in this document, we'll display dimensions as follows:
+- Dimensions known at ad-serving time on a green background.
+- Dimensions known at conversion time on a yellow background and underlined.
+{% endAside %}
+
+The dimensions you want to track for this example look like this:
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/7H0FNCjZM9zr3VE48Q2G.png", alt="Campaign ID, Geography ID, and product category.", width="498", height="55" %}
+
+### What are aggregation keys (buckets)?
+
+Aggregation key and bucket refer to the same thing. The term _aggregation key_ is used in the browser APIs used to configure reports. The term _bucket_ is used in the aggregatable and summary reports, and in the [aggregation service APIs](/docs/privacy-sandbox/aggregation-service/).
+
+An aggregation key (key for short) is a piece of data that represents the values of the dimensions being tracked. Data is later aggregated along each aggregation key.
+
+For example, let's assume you're tracking the dimensions Product category, Geography ID, and Campaign ID.
+
+When a user located in Geography ID 7 sees an ad for Campaign ID 12, and later converts by purchasing a product in Product category 25, you may set an aggregation key that looks like this:
+
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/dmpyjw7KIcunQemDqs8i.png", alt="Aggregation key for a conversion.", width="495", height="125" %}
+
+You'll see later that an aggregation key does not look exactly like this in practice, but for now let's focus on the information contained in the key.
+
+### What are aggregatable values?
+
+To answer your questions with the dimensions we've outlined, you want to know, across many users and many keys:
+
+-   The number of purchases (the purchase count). Once aggregated and made available in a summary report, this will be the total purchase count (summary value).
+-   The revenue for each purchase (the purchase value). Once aggregated and made available in a summary report, this will be the total revenue (summary value).
+
+Each of these—the purchase count for one conversion and the purchase value for one conversion—is an aggregatable value. You can think of aggregatable values as the values of your measurement goals.
+
+<table class="with-heading-tint">
+	<thead>
+		<tr><th rowspan="2">Question</th>
+			<th rowspan="2">Aggregatable value = Measurement goal
+			</th>
+        </tr>
+	</thead>
+	<tbody>
+		<tr>
+            <td>How many <strong>purchases</strong>…</td>
+			<td><strong>Purchase count</strong></td>
+		</tr>
+		<tr>
+			<td>How much <strong>revenue</strong>…</td>
+			<td><strong>Purchase value</strong></td>
+		</tr>
+	</tbody>
+</table>
+
+When a user located in Geography ID 7 sees an ad for Campaign ID 12, and later converts by purchasing a product of Product category 25 for $120, you may set an aggregation key and aggregatable values that look like these:
```

Comment:
I actually don't know what we do with currency symbols, but is this written assuming only US ad tech?